README.md

```diff
@@ -277,6 +278,24 @@ You can't access the MongoDB from outside the server. To access the MongoDB shel
 
 > Later on we'll be using a separate MongoDB instance for every app.
 
+### Using External Mongo Instances
+
+To not have a local docker instance of Mongo installed and run, simply comment
+out the `mongo` section in your `mup.js` config file:
```

Comment:
> simply comment out

Please change it to "simply remove or comment out"

Also, I'm not sure the example is needed. What is your opinion?